### PR TITLE
Adding load script syntax documentation

### PR DESCRIPTION
**Pull request by axelssonHakan**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *qlikcore* ready!

Built with commit c717f58b7e53fcf3689c4e43e89e800c98c2ce18

https://deploy-preview-350--qlikcore.netlify.com

File: docs/services/qix-engine/script_syntax/Script_regular_statements.md [hunk 1]
```diff
@@ -0,0 +1,1736 @@
...
+
+Direct Discovery does not support using aggregations in
+ **LOAD**
+statements. If aggregations are used, the results are unpredictable. A
+ **LOAD**
+statement such as the following should not be
+used:
+
+```code
+DIRECT QUERY DIMENSION stateid, SUM(amount\*7) AS MultiFirst MEASURE amount FROM sales_table;
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_syntax/Script_regular_statements.md]: The * should not be escaped

File: docs/services/qix-engine/script_syntax/financial_aggregation_functions.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+fields listed in the results column to a sheet in your app to see the
+result.
+
+### Example
+
+Script:
+
+```code
+Cashflow:
+LOAD 2013 as Year, \* inline [
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_syntax/financial_aggregation_functions.md]: The * should not be escaped

File: docs/services/qix-engine/script_reference/Script_regular_statements.md [hunk 3]
```diff
@@ -0,0 +1,1736 @@
...
+
+Direct Discovery does not support using aggregations in
+ **LOAD**
+statements. If aggregations are used, the results are unpredictable. A
+ **LOAD**
+statement such as the following should not be
+used:
+
+```code
+DIRECT QUERY DIMENSION stateid, SUM(amount\*7) AS MultiFirst MEASURE amount FROM sales_table;
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_reference/Script_regular_statements.md]: The * should not be escaped inside code blocks

File: docs/services/qix-engine/script_reference/financial_aggregation_functions.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+fields listed in the results column to a sheet in your app to see the
+result.
+
+### Example
+
+Script:
+
+```code
+Cashflow:
+LOAD 2013 as Year, \* inline [
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_reference/financial_aggregation_functions.md]: The * should not be escaped inside code blocks

File: docs/services/qix-engine/script_reference/mathematical_functions.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+`rand()`
+
+This example script creates a table of 1000 records with randomly
+selected upper case characters, that is, characters in the range 65 to
+91
+(65+26).
+
+```qlik
+Load Chr(Floor(rand() \* 26) + 65) as UCaseChar, RecNo() as ID
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_reference/mathematical_functions.md]: The * should not be escaped inside a codeblock

File: docs/services/qix-engine/script_reference/string_aggregation_functions.md [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+<table>
+<thead>
+<tr><th>Example</th><th colspan='2'>Result</th></tr>
+</thead>
+<tbody>
+<tr>
+    <td>
+    <code>
+        TeamData:<br>
+        LOAD \* inline [<br>
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_reference/string_aggregation_functions.md]: The * should not be escaped inside a code block

File: docs/services/qix-engine/script_reference/Script_regular_statements.md [hunk 7]
```diff
@@ -0,0 +1,1736 @@
...
+Cd</br>
+Ef</br>
+Gh</br>
+</code>
+<br>All values are capitalized.</br></td>
+</tr>
+<td><br>This example shows how to force case upper.</br>
+<code>
+FORCE Case Upper;</br>
+CaseUpper:</br>
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_reference/Script_regular_statements.md]: the double * does not make it bold, I think you should use the html tags <b>

File: docs/services/qix-engine/script_reference/Script_regular_statements.md [hunk 8]
```diff
@@ -0,0 +1,1736 @@
...
+LOAD * Inline [</br>
+ab</br>
+Cd</br>
+eF</br>
+GH</br>
+];</br></td>
+<td><br>The  **CaseUpper**  table contains the following values:</br>
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_reference/Script_regular_statements.md]: the double * does not make it bold, I think you should use the html tags <b>

File: docs/services/qix-engine/script_reference/Script_regular_statements.md [hunk 9]
```diff
@@ -0,0 +1,1736 @@
...
+<br>FORCE Case Lower;</br>
+CaseLower:</br>
+LOAD * Inline [</br>
+ab</br>
+Cd</br>
+eF</br>
+GH</br>
+];</br></td>
+</code>
+<td><br>The  **CaseLower**  table contains the following values:</br>
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_reference/Script_regular_statements.md]: the double * does not make it bold, I think you should use the html tags <b>

File: docs/services/qix-engine/script_reference/Script_regular_statements.md [hunk 10]
```diff
@@ -0,0 +1,1736 @@
...
+<br>FORCE Case Mixed;</br>
+CaseMixed:</br>
+LOAD * Inline [</br>
+ab</br>
+Cd</br>
+eF</br>
+GH</br>
+];</br></td>
+</code>
+<td><br>The  **CaseMixed**  table contains the following values:</br>
```
Comment (FredrikFolkesson) [inline docs/services/qix-engine/script_reference/Script_regular_statements.md]: the double * does not make it bold, I think you should use the html tags <b>

**Comment by Johnken-S** [comment]
unescape * and bold font ones should be fixed now

**Pull request merged**